src/pages/kyunghyun/Main/feed/FeedInfo.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import CommentForm from '../comment/CommentForm';
+import CommentList from '../comment/CommentList';
+import './feed.scss';
+
+class FeedInfo extends Component {
+  id = 3;
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/commentData.json')
+      .then(res => res.json())
+      .then(res => {
+        this.setState({
+          information: res.C1,
+        });
+      });
+  } // json파일 패치하기
+
+  state = {
+    information: [], // 댓글 데이터가 들어갈 배열
+  };
+
+  handleCreate = data => {
+    const { information } = this.state;
+    this.setState({
+      information: information.concat(
+        Object.assign({}, data, {
+          id: this.id++,
+        }) // 비어있는 객체{}에 data를 집어넣고,   id: this.id++,도 집어넣기 or ...data, id:  this.id++; 해도 됨 = 불변성 유지
+      ),
+    });
+  }; // 댓글 데이터 생성하는 함수 : data (CommentForm의 state.comment) 를 파라미터로 받아 concat/Object.assign으로 생성한 새로운 information배열을 가지고, setState로 State변경
+
+  handleRemove = id => {
+    const { information } = this.state;
+    this.setState({
+      information: information.filter(info => info.id !== id),
+      // info.id가 파라미터로 일치하지 않는 원소만 추출해서 새로운 배열을 만듬
+      // 선택한 'id'를 제외한 나머지 id들로 새로운 배열을 만들어서 표시함
+      // info.id 가 id 인 것을 제거함
+    });
+  };
+
+  handleUpdate = (id, data) => {
+    const { information } = this.state;
+    this.setState({
+      information: information.map(info => {
+        if (info.id === id) {
+          return {
+            id,
+            ...data,
+          };
+        }
+        return info;
+      }),
+    });
+  }; // 배열 수정하는 함수 : id 와 data(를 파라미터로 받아 id가 매치되는 배열의 data를 map함수를 사용해 새로운 data로 변환시킴
+  render() {
+    const { id, mainImg, profileImg } = this.props.feed;
+    const { information } = this.state;
+    return (
+      <>
+        <article key={id}>
```

Comment:
```suggestion
      <article>
```
- fragment 불필요해요
- key는 반복문 때문에 children을 비교할 필요가 있을 때 사용합니다 여기서는 불필요하고 공식 문서 다시 확인해주세요